scipy/special/specfun/specfun.f

```diff
@@ -21,6 +21,9 @@
 C         - SF_ERROR_DOMAIN    = 7: out of domain
 C         - SF_ERROR_ARG       = 8: invalid input parameter
 C         - SF_ERROR_OTHER     = 9: unclassified error
+C       - Changed initial guesses for roots in JYZO.
+C         Good guesses found by fitting a cos-function to the Bessel
+C         function and using it to predict the next root.
```

Comment:
```suggestion
C       - Improved initial guesses for roots in JYZO.
```

Fitting a cos-function isn't the full story for where those expansions come from. I think it's better to leave the details out here. Also, you didn't just change the guesses, you improved them, good to say explicitly.

Reply:
Thanks, I've changed the comment.

If there is an effort to rewrite the module at some point I'd be happy to participate.

Reply:
> If there is an effort to rewrite the module at some point I'd be happy to participate.

That would be much appreciated. @tirthasheshpatel and have a small development grant to work on the `special` infrastructure and will be making a push to streamline special function development. We hope to eventually replace all of the buggy Fortran 77 in `special`. I’m going to create a tracking issue for `special` within the next couple weeks. I’ll ping you.